src/test/java/sk/uniba/fmph/dcs/game_phase_controller/FeedTribeStateTest.java

```diff
@@ -0,0 +1,87 @@
+package sk.uniba.fmph.dcs.game_phase_controller;
+
+import org.junit.Test;
+import sk.uniba.fmph.dcs.stone_age.*;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+
+import static org.junit.Assert.assertEquals;
+
+public class FeedTribeStateTest {
+
+    class InterfaceFeedTribeMock implements InterfaceFeedTribe {
+        private boolean returnValue;
+
+        InterfaceFeedTribeMock(boolean returnValue) {
+            this.returnValue = returnValue;
+        }
+
+        @Override
+        public boolean feedTribeIfEnoughFood() {
+            return returnValue;
+        }
+
+        @Override
+        public boolean feedTribe(Collection<Effect> resources) {
+            return returnValue;
+        }
+
+        @Override
+        public boolean doNotFeedThisTurn() {
+            return returnValue;
+        }
+
+        @Override
+        public boolean isTribeFed() {
+            return returnValue;
+        }
+    }
+
+    @Test
+    public void test_tryToMakeAutomaticAction() {
+
+        HashMap<PlayerOrder, InterfaceFeedTribe> map = new HashMap<>();
+        PlayerOrder player1 = new PlayerOrder(1, 1);
+        PlayerOrder player2 = new PlayerOrder(2, 2);
+
+        map.put(player1, new InterfaceFeedTribeMock(false));
+        map.put(player2, new InterfaceFeedTribeMock(true));
+
+        FeedTribeState feedState = new FeedTribeState(map);
+        assertEquals(feedState.tryToMakeAutomaticAction(player1), HasAction.WAITING_FOR_PLAYER_ACTION);
+        assertEquals(feedState.tryToMakeAutomaticAction(player2), HasAction.NO_ACTION_POSSIBLE);
```

Comment:
Chcelo by to dat testy aj na `AUTOMATIC_ACTION_DONE`. Tu mock classu by som asi skor spravil tak, ze na zaklade poctu jedla / figurok rozhodne. Potom sa budu lepsie pisat tie testy

Reply:
fixed it